src/Core/Managed/Shared/Extensibility/Implementation/JsonSerializer.cs

```diff
@@ -45,9 +45,11 @@ public static byte[] Serialize(IEnumerable<ITelemetry> telemetryItems, bool comp
         {
             var memoryStream = new MemoryStream();
             using (Stream compressedStream = compress ? CreateCompressedStream(memoryStream) : memoryStream)
-            using (var streamWriter = new StreamWriter(compressedStream, TransmissionEncoding))
             {
-                SeializeToStream(telemetryItems, streamWriter);
+                using (var streamWriter = new StreamWriter(compressedStream, TransmissionEncoding))
```

Comment:
The previous sequential usings has the same semantic meaning as this nesting syntax so this change is not needed.